RockLib.Configuration.Remote/JsonConfigurationParser.cs

```diff
@@ -0,0 +1,77 @@
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Newtonsoft.Json;
```

Comment:
Could we use `System.Text.Json`?

Reply:
Yes, I'll try to swap this out.

Reply:
Updated, it was a pretty smooth transition.

Only notable difference is that `JsonValue.ToString()` for boolean returns it in all lowercase (e.g. `true`). Newtonsoft's `JToken.ToString()` used initial uppercase (e.g. `True`).

This _could_ be a breaking change for anyone who's reading config values directly instead of binding into a class, and using case-sensitive comparison against `True` and `False`. I think it's unlikely, but definitely a possibility.